samples/sensor/explorir_m/src/main.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023, Vitrolife A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(MAIN);
+
+int main(void)
+{
+	struct sensor_value value;
+	const struct device *const dev = DEVICE_DT_GET_ANY(gss_explorir_m);
+
+	if (!device_is_ready(dev)) {
+		LOG_ERR("%s is not ready", dev->name);
+		return 0;
+	}
+
+	while (1) {
+		if (sensor_sample_fetch(dev) == 0 &&
+		    sensor_channel_get(dev, SENSOR_CHAN_CO2, &value) == 0) {
```

Comment:
Please convert this sample application into a more generic one that can be used with any CO2 sensor. See `samples/sensor/accel_polling` for example

Reply:
Good idea. I've changed the sample.

An even more generic polling sample might be useful which can be used with any type of sensor.
Then maybe only a dts overlay would be required. Not sure how to map which channel is supported by the sensor though.